fe1-web/src/features/popcha/interface/Configuration.ts

```diff
@@ -0,0 +1,17 @@
+import { Hash } from 'core/objects';
+import FeatureInterface from 'core/objects/FeatureInterface';
+
+import { PoPchaFeature } from './Features';
+
+export const POPCHA_FEATURE_IDENTIFIER = 'popcha';
+
+export interface PoPchaConfiguration {
+  useCurrentLaoId: () => Hash;
```

Comment:
For now there is only one function that is defined in PoPchaConfiguration so this is not really an issue but I would recommend to get used to add complete documentation to each element of the configuration, as we did in other features.